src/elements/TextInput/TextInput.tsx

```diff
@@ -12,7 +12,7 @@ import React, { FC, FormEvent, KeyboardEvent, ReactElement, ReactNode, useEffect
 export enum TextInputType {
     Text = "text",
     Password = "password",
-    Number = "number",
+    Number = "numeric",
```

Comment:
`number` is the correct type, no?
If this to remove the number increment arrows then I would go with CSS
https://www.w3schools.com/howto/howto_css_hide_arrow_number.asp

Reply:
another way would be to use the React TypeScript definition for the input type attribute:

```
type?: InputHTMLAttributes<HTMLInputElement>["type"];
```

(in the prop types, rather than enum)

Reply:
@deniaz IMO we should not go with this solution as everything related to styles or sizes already is an `enum`, it wouldn't make sense to have this one as `string`.